CRITICAL - 2020-11-21 05:59:55 --> Too few arguments to function App\Models\Admin\Delivery_data_model::saveDelivery(), 0 passed in C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php on line 529 and exactly 1 expected
#0 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(529): App\Models\Admin\Delivery_data_model->saveDelivery()
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#3 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-21 06:00:11 --> Call to undefined method App\Models\Admin\Delivery_data_model::insertID
#0 C:\xampp\htdocs\internal-nutrilo\app\Models\Admin\Delivery_data_model.php(21): CodeIgniter\Model->__call('insertID', Array)
#1 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(529): App\Models\Admin\Delivery_data_model->saveDelivery(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#3 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2020-11-21 06:00:37 --> Call to undefined method App\Models\Admin\Delivery_data_model::insertID
#0 C:\xampp\htdocs\internal-nutrilo\app\Models\Admin\Delivery_data_model.php(21): CodeIgniter\Model->__call('insertID', Array)
#1 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(529): App\Models\Admin\Delivery_data_model->saveDelivery(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#3 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2020-11-21 06:12:57 --> Cannot use object of type CodeIgniter\Database\MySQLi\Result as array
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->delivery()
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2020-11-21 08:41:23 --> array_keys() expects parameter 1 to be array, null given
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'array_keys() ex...', 'C:\\xampp\\htdocs...', 618, Array)
#1 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(618): array_keys(NULL)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->delivery()
#3 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2020-11-21 08:42:03 --> array_keys() expects parameter 1 to be array, null given
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'array_keys() ex...', 'C:\\xampp\\htdocs...', 621, Array)
#1 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(621): array_keys(NULL)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->delivery()
#3 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2020-11-21 14:31:38 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(554): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 14:32:42 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(554): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 14:33:52 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(554): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 14:34:14 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(556): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 14:37:11 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(555): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 14:37:17 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(555): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 14:41:43 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(555): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 14:58:54 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(555): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 14:59:35 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(441): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(558): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:00:08 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(559): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:00:27 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:00:46 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:00:53 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:01:13 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(561): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:02:48 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(561): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:03:19 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(561): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:05:49 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(562): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:07:13 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(562): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:07:59 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(562): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:09:31 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(562): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:11:57 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(562): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:12:06 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(562): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:12:31 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(442): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(562): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:12:59 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(556): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:15:17 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(556): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:22:41 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(556): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.549852,98.698...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 15:28:05 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(559): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 15:39:36 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(559): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 15:39:41 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(559): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 15:44:17 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(559): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:13:15 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(559): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:13:49 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(561): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:14:18 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(561): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:18:10 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(561): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:20:29 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(559): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:24:15 --> Call to a member function request() on null
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2020-11-21 16:24:32 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(559): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:27:30 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:32:33 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:32:36 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:32:38 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:36:00 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:36:28 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:39:56 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:40:17 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(557): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-21 16:41:00 --> 22 : The requested URL returned error: 400 Bad Request
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(821): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested U...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(449): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\HTTP\CURLRequest.php(158): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://maps.go...')
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(438): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://maps.go...')
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(556): App\Controllers\Admin\Driver->getTimeDelivery('3.580994,98.662...', '3.509716,98.668...')
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->startDriving()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-11-21 16:59:48 --> key() expects parameter 1 to be array, null given
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'key() expects p...', 'C:\\xampp\\htdocs...', 212, Array)
#1 C:\xampp\htdocs\internal-nutrilo\app\Views\Admin\Driver_deliver_view.php(212): key(NULL)
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('Admin/Driver_de...', Array, true)
#4 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(651): view('Admin/Driver_de...', Array)
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->delivery()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#7 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
